lib/l10n/app_fr.arb

```
@@ -190,5 +190,14 @@
   "signerErrorRequestInProgress": "Une autre requête est en cours. Veuillez patienter.",
   "signerErrorNoActivity": "Impossible de lancer le signer. Veuillez réessayer.",
   "signerErrorLaunchError": "Échec du lancement de l'application signer.",
-  "signerErrorUnknown": "Une erreur s'est produite avec le signer. Veuillez réessayer."
+  "signerErrorUnknown": "Une erreur s'est produite avec le signer. Veuillez réessayer.",
+  "carouselPrivacyTitle": "Confidentialité et sécurité",
+  "carouselPrivacyDescription": "Gardez vos conversations privées. Même en cas de violation, vos messages restent sécurisés.",
+  "carouselIdentityTitle": "Choisissez votre identité",
+  "carouselIdentityDescription": "Discutez sans révéler votre numéro de téléphone ou votre e-mail. Choisissez votre identité : nom réel, pseudonyme ou anonyme.",
+  "carouselDecentralizedTitle": "Décentralisé et sans permission",
+  "carouselDecentralizedDescription": "Aucune autorité centrale ne contrôle votre communication – pas de permissions nécessaires, pas de censure possible."
+,
```

RuboCop (1.84.0)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)